README.md

```diff
@@ -102,5 +103,6 @@ git pull origin master
 git tag -a v$NEW_VERSION -m ''
 git tag -a grpc/go-cubic/v$NEW_VERSION -m ''
 git tag -a grpc/go-cubic/cubicpb/gw/v$NEW_VERSION -m ''
+git tag -a grpc/swift-cubic/v$NEW_VERSION -m ''
```

Comment:
If we don't need the grpc/swift-cubic tag, we can leave it out. For example, Python only uses the v$NEW_VERSION tag when fetching using pip.

Reply:
I've removed it.